docs/source/multi_gpu.rst

```diff
@@ -202,10 +202,20 @@ Note in particular the difference between `gpus=0`, `gpus=[0]` and `gpus="0"`.
 
 .. note::
 
-    When specifying number of gpus as an integer `gpus=k`, setting the trainer flag
-    `auto_select_gpus=True` will automatically help you find `k` gpus that are not
+    When specifying number of gpus as an integer `gpus=k`, setting the trainer flag 
+    `auto_select_gpus=True` will automatically help you find `k` gpus that are not 
     occupied by other processes. This is especially useful when GPUs are configured
-    to be in "exclusive mode", such that only one process at a time can access them.
+    to be in “exclusive mode”, such that only one process at a time can access them.
```

Comment:
just some minor formatting here:
- remove the added trailing spaces at the end of lines 205, 206
- revert the quotations “exclusive mode” to "exclusive mode"